admin/links/class-link-type-classifier.php

```diff
@@ -58,11 +58,11 @@ public function classify( $link ) {
 	}
 
 	/**
-	 * Returns true when the link starts with https:// or http://
+	 * Checks whether a link starts with a HTTP[S] protocol.
```

Comment:
@jrfnl Since the H in `HTTPS` is pronounced "age", it should be `an HTTP[S] protocol`.

Reply:
Fixed x 2